frontend/src/components/Layout/Footer.tsx

```diff
@@ -11,6 +11,17 @@ export const Footer = () => {
         <span className="text-sm text-secondary-500 dark:text-secondary-300 sm:text-center">
           © 2023 PermaplanT
         </span>
+        <span className="text-sm text-secondary-500 dark:text-secondary-300 sm:text-center">
+          <a
+            href="https://github.com/ElektraInitiative/PermaplanT"
+            style={{ display: 'flex', alignItems: 'center' }}
+          >
+            <svg viewBox="0 0 1024 1024" fill="currentColor" height="1em" width="1em">
```

Comment:
Storing the svg in a file and importing it makes the markup more clean and the svg reusable.
You can look at the icons we use in different places as a reference.